numbat/modules/math/geometry.nbt

```diff
@@ -2,9 +2,11 @@ use core::functions
 use math::constants
 
 @description("The length of the hypotenuse of a right-angled triangle $\\sqrt\{x^2+y^2\}$.")
+@example("hypot2(3, 4)")
```

Comment:
One nice thing is that `hypot2` (and 3) take dimensionful arguments, so maybe we could use
```suggestion
@example("hypot2(3 m, 4 m)")
```
here?